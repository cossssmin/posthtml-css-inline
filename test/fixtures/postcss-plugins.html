<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px !important;
    }

    @media (min-width: 640px) {
      .text-sm {
        font-size: 16px !important;
      }
    }
  </style>
</head>
<body>
  <div class="text-sm">text</div>
</body>
</html>
