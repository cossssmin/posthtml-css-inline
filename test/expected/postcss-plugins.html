<!DOCTYPE html>
<html>
<head>
  <style>
    .text-sm {
      font-size: 12px;
    }

    @media (min-width: 640px) {
      .text-sm {
        font-size: 16px;
      }
    }
  </style>
</head>
<body>
  <div class="text-sm" style="font-size: 12px">text</div>
</body>
</html>
